### ISSUE TITLE
Bug: change accessibility tags to Russian language

### ISSUE THREAD

**Issue by wwily** [p-foundation/pink-app#45]
**Describe the bug**
Our project Pink is in Russian language. This implies that all content in it should be in Russian, including attributes for accessibility and title. Our pages `index.html` and `components.html` have their language set to English within the HTML tag. You can learn more about the tag [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang?retiredLocale=uk#accessibility_concerns).

```html
<html lang="en">
```

Also the title of our `index.html` is "Pink App". We ought to interpret it into "Приложение Pink". We can leave the word "Pink" in English because it is the proper name. At that point we got to ensure that all of the words "Pink" in our project are in English. (At least the title of page `form.html` has the word "Пинк", but it may be more, go through all HTML files and check if all texts are in Russian).
**Hint**: take a look at the section _reviews_ and its _aria-labels_.

To summarize, we have:
- changing `lang` attribute to `ru`
- changing the `title` of `index.html` to the right one
- replacing all "Пинк" with "Pink"
- reviewing all HTML files and replacing all texts and attributes for accessibility with Russian language.

**To Reproduce**
1. Go to `index.html`, `components.html`
2. Look at the lang tag and attributes for accessibility.
3. See that some text in the attributes for accessibility are written in English. We can also see that the attribute `lang` in tag `<html>` does not have the appropriate value.

**Expected behavior**
When the bug is fixed, the lang tag will be changed in `index.html` and `components.html`. All accessibility attributes are in Russian, while leaving the name "Pink" in English. 

**Desktop:**
 - OS: **Windows**
 - Browser: **Сhrome**
 - Version: **123.0.6312.59**

**Quick Instruction for Working on the Issue**
‼️ Before checking out to a new branch, ensure you have the latest updates from the main by running `git pull` command.
- **Please create a new branch**, and name it `fix-accessibility-tags`
- **Make Changes:** address the issue within the branch
- **Test thoughtfully:** go to the site, open the developer tools and click `Accessibility` and check elements.

![image](https://github.com/p-foundation/pink-app/assets/126797593/08f85a14-b017-4169-a2fb-0f62e443b28e)

- **Open a pull request:** reference the specific issue in your PR description using the following format:

`This pull request resolves [#issue_number](link_to_the_issue)`. Replace `issue_number` with the actual issue number and `link_to_the_issue` with the link to the respective issue. For example:

```bash
This pull request resolves [#123](https://github.com/username/repository/issues/123)
```

**Comment by MarPostovik** [comment]
Hi @wwily . Good job on writing issue! I have a few suggestions to improve it:
- We don't have accessibility tags. They are called attributes for accessibility. 
- Here don't write "a person who will...". Just write: "Go through all HTML files and check if all texts are in Russian":
> (At least the title of page form.html has the word "Пинк", but it may be more, a person who will solve the issue will need to check).
- Let's divide the issue into a few subtasks to organize it better:
  - changing `lang` attribute to `ru`
  - changing the `title` of `index.html` to the right one
  - replacing all "Пинк" with "Pink"
  - reviewing all HTML files and replacing all texts and attributes for accessibility with Russian language.
- In the part **Test thoughtfully** provide the way to test if the issue is fixed correctly.

**Comment by wwily** [comment]
Fixed)

**Comment by NazarPauk** [comment]
.take
